src/Controllers/NotFoundRedirectController.cs

```diff
@@ -272,10 +272,10 @@ public FileUploadJsonResult ImportRedirects(HttpPostedFileBase xmlfile)
             // Save all redirects from xml file
             CustomRedirectCollection redirects = parser.Load();
             string message;
-            if (redirects != null || redirects.Count != 0)
+            if (redirects != null || redirects.Count() != 0)
```

Comment:
`if(redirects?.Any())` ?

Reply:
This is legacy code which will be refactored later. Cannot fix everything at once :)